docs/adr/0008-refrain-from-using-docker-useremulation-in-nextflow.md

```diff
@@ -0,0 +1,23 @@
+# 8. Refrain from using docker userEmulation in Nextflow
+
+Date: 2022-06-02
+
+## Status
+
+Accepted
+
+## Context
+
+There were multiple discussions over  the time around whether to utilize the `userEmulation` feature in Nextflow when using the `docker` scope as this both resolves issues with users running docker as root as well as causes issues for users with running any Spark based tool. 
```

Comment:
For context, should we also mention `docker.fixOwnership` and `docker.runOptions` here?